src/pykiso/test_coordinator/assert_step_report/report_template.html.j2

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
I think you have to update the pyproject.toml with this file and especially the include tag

Reply:
Normally with poetry all files present in the package, python or not, are included by default
But it's still more explicit to list them all in the pyproject.toml

Reply:
Done